lib/travis/api/v3/models/cron.rb

```diff
@@ -81,8 +81,8 @@ def always_run?
     end
 
     def last_non_cron_build_time
-      last_build = Build.find_by_id(branch.last_build_id)
-      last_build.finished_at.to_datetime.utc if last_build && last_build.finished_at
+      return unless last_build = Build.find_by_id(branch.last_build_id)
+      last_build&.started_at.to_datetime.utc
```

Comment:
we could probably even skip the `return unless` and do `last_build&.started_at&.to_datetime.utc`?